frontend/pages/index.tsx

```diff
@@ -22,6 +46,19 @@ const HomePage = () => {
       <h2>Welcome to Republic of Rome Online</h2>
       <p><i>Experience the intrigue and power struggles of Ancient Rome, right from your browser</i></p>
 
+      <h3 id="waitlist">Join our waitlist</h3>
+        <form onSubmit={handleSubmit}>
+          <Stack alignItems="start" spacing={2}>
+            <TextField required
+                id="email"
+                label="Email"
+                onChange={handleEmailChange}
+                helperText={"johndoe@example.com"}
+                style={{width: "300px"}} />
+            <Button variant="contained" type="submit">Join waitlist</Button>
+          </Stack>
+        </form>
```

Comment:
Could you move the waitlist to under the Early Development Notice and Wiki, but above the Exclusive Features bit?

Reply:
I've done it and think it's a good choice of yours. It now doesn't distract too much from the Early Development Notice and the Wiki.